topostats/filters.py

```diff
@@ -296,7 +297,7 @@ def gaussian_filter(self, image: np.ndarray, **kwargs) -> np.array:
             **kwargs,
         )
 
-    def filter_image(self) -> None:
+    def filter_image(self, scars_config: dict) -> None:
```

Comment:
If the `scars_config` section is going to remain as an independent top-level dictionary in `default_config.yaml` then this will be required.

But there is the option of moving the whole `remove_scars` configurations and nesting it under the `filter:` section.  If this is going to be done then you wouldn't need to pass `scars_config` in as a separate argument to `process_scan` as it is nested within `filter_config`.